phpunit.xml

```diff
@@ -1,5 +1,4 @@
 <phpunit bootstrap="tests/bootstrap.php"
```

Comment:
the phpunit.xml file should be renamed to phpunit.xml.dist and phpunit.xml be in .gitignore. then its much easier to customize the phpunit file.